docs/pages/kubernetes-access/helm/guides/aws.mdx

```diff
@@ -532,6 +534,54 @@ Here's an example where we set the chart to use 3 replicas:
   To change `chartMode`, `clusterName`, or any `aws` settings, you must first uninstall the existing chart and then install a new version with the appropriate values.
 </Admonition>
 
+### Autoscaling
+
+In order to reduce DynamoDB costs you might want to enable DynamoDB autoscaling.
+This can be done manually or delegated to Teleport.
+
+This is step is usually done after a successful Teleport deployment, once you have gathered some data
+about Teleport's DynamoDB usage and know what a regular usage looks like and how autoscaling should be tuned.
+See [the DynamoDB autoscaling section](../../../setup/reference/backends.mdx#dynamodb-autoscaling) for more details.
+
+To go further you have to know the desired read/write minimum, maximum and target capacity.
+If you decide to let Teleport set up the autoscaling proceed as follow:
+
+<Tabs>
+  <TabItem label="Using values.yaml">
+  Set the following fields in your existing `aws-values.yaml` file and replace the numeric values by yours:
+
+  ```yaml
+  aws:
+    # [...] already present values under `aws`
+    autoScaling: true
+    readMinCapacity: 0     # integer
+    readMaxCapacity: 1     # integer
+    readTargetValue: 0.5   # float
+    writeMinCapacity: 0    # integer
+    writeMaxCapacity: 1    # integer
+    writeTargetValue: 0.5  # float
+  ```
```

Comment:
I'd suggest setting these way higher than currently just because if anyone copy/pastes these values, they'll find their cluster doesn't work.

5-100 is usually a pretty good autoscaling range and the target utilization probably wants to be around 50% - this should give sane defaults for anyone who's just casually setting these values with no real idea of what they affect (it happens way more often than you'd think).

Teleport's use of DynamoDB is very bursty so 50% is sometimes not enough either, but obviously if we set it to 20% all the time then it's going to cost people more.

```suggestion
  aws:
    # [...] already present values under `aws`
    autoScaling: true
    readMinCapacity: 5     # integer
    readMaxCapacity: 100   # integer
    readTargetValue: 50.0  # float
    writeMinCapacity: 5    # integer
    writeMaxCapacity: 100  # integer
    writeTargetValue: 50.0 # float
  ```
```